setup.py

```diff
@@ -24,17 +25,26 @@ def get_version():
     raise RuntimeError('Unable to find version string in %s.' % (file_path,))
 
 
-def load_requirements(*requirements_paths):
+def load_requirements(requirements_folder):
     """
     Load all requirements from the specified requirements files.
     Returns a list of requirement strings.
     """
+    edx_platform_version = get_edx_platform_version()
+
+    if edx_platform_version == 0.9:
+        requirements_path = '{}/{}'.format(requirements_folder, 'ginkgo.in')
+    elif edx_platform_version == 0.10:
```

Comment:
Same comment here, 0.10 should be a constant, such as: OPEN_EDX_HAWTHORN_VERSION = 0.10. @andrey-canon